jobserver/views/workspaces.py

```diff
@@ -226,24 +226,11 @@ def get(self, request, *args, **kwargs):
             context=context,
         )
 
-    def get_output_permissions(self, user, workspace):
-        # a user can see backend files if they have access to at least one
-        # backend and the permissions required to see outputs
-        is_privileged_user = has_permission(
-            user, permissions.release_file_view, project=workspace.project
-        )
-        has_backends = (
-            user.is_authenticated and user.backends.exclude(level_4_url="").exists()
-        )
-        can_view_files = is_privileged_user and has_backends
-
+    def get_output_permissions(self, workspace):
         # are there any releases to show for the workspace?
         can_view_releases = workspace.releases.exists()
 
         return {
-            "level_4": {
-                "disabled": not can_view_files,
-            },
```

Comment:
question: I believe this is safe, because of 438b49cbb44739a582e51756560a40706d65e267 which removed the use of `outputs.level_4.disabled` and there is no current use of this part of the context otherwise in the template. Is my reasoning correct?

(It seems worth double checking that this change is correct.)